### ISSUE TITLE
[Sub-Epic] UX & Docs

### ISSUE THREAD

**Issue by marisamorby** [gatsbyjs/gatsby#6820]
This sub-epic contains issues related to UX research and docs for Gatsby v2 Release.

**Comment by shannonbux** [comment]
Am I understanding these things correctly, @KyleAMathews and @calcsam ?
- When v2 goes stable, we’ll take down the banners that link next.gatsbyjs.org and www.gatsbyjs.org
- Policy is to update v2 docs only and not focus on v1 docs (we are not committed to maintaining them right now). Is this correct @KyleAMathews and @calcsam?
- Once v2 goes stable, we'll push next.gatsbyjs.org to www.gatsbyjs.org and delete next.gatsbyjs.org. www.gatsbyjs.org will be the home of v2 docs, and v1 docs will live _______ (where?)

**Comment by calcsam** [comment]
@shannonbux yes -- let's keep this discussion over at https://github.com/gatsbyjs/gatsby/pull/6843#issuecomment-409047968?

**Comment by shannonbux** [comment]
@calcsam did you mean to link to the READme v2 edit? Maybe I'm missing the comment there that's relevant. I want to keep some instructions on this sub-epic so that people making changes to the docs know where to make them, so I could also just hyperlink to wherever the clearest instructions/discussion is!
